DXMainClient/DXGUI/Multiplayer/CnCNet/LoadSaveGameOptionsWindow.cs

```diff
@@ -0,0 +1,245 @@
+
+            var margin = 10;
+
+            lblHeader = new XNALabel(WindowManager);
+            lblHeader.FontIndex = 1;
+            lblHeader.ClientRectangle = new Rectangle(
+                margin, margin,
+                150, 22
+            );
+
+            var lblPresetName = new XNALabel(WindowManager);
+            lblPresetName.Text = "Preset Name";
+            lblPresetName.ClientRectangle = new Rectangle(
+                margin, lblHeader.Bottom + margin,
+                150, 18
+            );
+
+            ddiCreatePresetItem = new XNADropDownItem();
+            ddiCreatePresetItem.Text = "[Create New]";
+
+            ddiSelectPresetItem = new XNADropDownItem();
+            ddiSelectPresetItem.Text = "[Select Preset]";
+            ddiSelectPresetItem.Selectable = false;
+
+            ddPresetSelect = new XNAClientDropDown(WindowManager);
+            ddPresetSelect.ClientRectangle = new Rectangle(
+                10, lblPresetName.Bottom + 2,
+                150, 22
+            );
+            ddPresetSelect.SelectedIndexChanged += DropDownPresetSelect_SelectedIndexChanged;
+
+            lblNewPresetName = new XNALabel(WindowManager);
+            lblNewPresetName.Text = "New Preset Name";
+            lblNewPresetName.ClientRectangle = new Rectangle(
+                margin, ddPresetSelect.Bottom + margin,
+                150, 18
+            );
+
+            tbNewPresetName = new XNATextBox(WindowManager);
+            tbNewPresetName.ClientRectangle = new Rectangle(
+                10, lblNewPresetName.Bottom + 2,
+                150, 22
+            );
+            tbNewPresetName.TextChanged += (sender, args) => RefreshLoadSaveBtn();
+
+            btnLoadSave = new XNAClientButton(WindowManager);
+            btnLoadSave.LeftClick += BtnLoadSave_LeftClick;
+            btnLoadSave.ClientRectangle = new Rectangle(
+                margin,
+                Height - UIDesignConstants.BUTTON_HEIGHT - margin,
+                UIDesignConstants.BUTTON_WIDTH_92,
+                UIDesignConstants.BUTTON_HEIGHT
+            );
+
+            var btnCancel = new XNAClientButton(WindowManager);
+            btnCancel.Text = "Cancel";
+            btnCancel.ClientRectangle = new Rectangle(
+                btnLoadSave.Right + margin,
+                btnLoadSave.Y,
+                UIDesignConstants.BUTTON_WIDTH_92,
+                UIDesignConstants.BUTTON_HEIGHT
+            );
+            btnCancel.LeftClick += (sender, args) => Disable();
+
+            AddChild(lblHeader);
+            AddChild(lblPresetName);
+            AddChild(ddPresetSelect);
+            AddChild(lblNewPresetName);
+            AddChild(tbNewPresetName);
+            AddChild(btnLoadSave);
+            AddChild(btnCancel);
+
+            Disable();
+        }
+
+        /// <summary>
+        /// Show the window
+        /// </summary>
+        /// <param name="isLoad">The "mode" for the window: load vs save</param>
```

Comment:
Summary and function parameters comments should be ended with punctuation marks like in the rest of the client code (or at least the most of it).